### ISSUE TITLE
Problems with Reading comments in MS Word

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4335]
**Reported by netblue on 2014-07-28 05:56**
Hi all, when using the keystroke NVDA+alt+c with the left alt key. NVDA, does not read the comment until the keystroke is released. This happens with all the language configurations on my keyboard. However when using the right alt key it works fine. This is a serious problem because a lot of Spanish speakers use NVDA and the right alt key is not available for this function on Spanish keyboard configurations.

**Comment by ehollig** [comment]
Closing, as no one is able to reproduce this issue, we are unable to contact the original poster, as far as I am aware of, and the issue that may of caused this has been resolved in snapshots as stated in https://github.com/nvaccess/nvda/issues/4335#issuecomment-155323156
